src/components/organisms/FAQItemBlock/styles.tsx

```diff
@@ -1,6 +1,15 @@
 import styled from "styled-components"
+import { space } from "../../../styles/vars"
+import { media } from "../../../styles/breakpoints"
+
 
 export const StyleFAQBlock = styled.div`
     margin: 0.5rem auto;
-    width: 600px;
+    max-width: 90vw;
+    width: 100%;
+
+    ${media.med`
+        margin: ${space.s} ${space.l};
+        width: 600px;
```

Comment:
The `600px`is for the container? This needs to be specified on Gatsby, not in the component.  
The component needs to be dumb as possible, avoid defining width. Only if it's really necessary. 

For containers is safer to use `max-width` instead `width`.

Reply:
Yeah, this is for the container. What do you mean about specifying the width in Gatsby though?

Reply:
When we start migrating the components to Gatsby pages (now we're working on Storybook only)  we'll create a page called "faq.tsx" and we're going to wrap the whole page with a container. So usually, you define the container in a `Page` component, not `Organism` component. 

Storybook helps you to work with a component in isolation, so we need to avoid specifying width or height. What if we want to use this component on a different page with a different width? Yes, we can add `props.width` but won't be easier to specify the width in a `Page` component rather than an `Organism` component?

Reply:
Oh, ok. That makes a lot of sense! I'll try to avoid specifying width from now on.